wallet/bindings/python/Cargo.toml

```diff
@@ -12,6 +10,10 @@ repository = "https://github.com/iotaledger/wallet.rs"
 license = "Apache-2.0"
 keywords = [ "iota", "wallet", "transaction", "python" ]
 categories = [ "cryptography::cryptocurrencies" ]
+publish = false
+
+[package.metadata.maturin]
+name = "iota-wallet"
```

Comment:
in iota.rs we just removed it and changed it to `[pyproject.toml](https://github.com/iotaledger/iota.rs/pull/1629/files#diff-6ae29472b05324aec56b7d87be8614015aaf98b1c3ead751da069830381cf8eb)`. 
```
[project]
name = "iota-client"
```

Reply:
@Thoralf-M were you still investigating something? You said my PR wasn't working

Reply:
iota.rs is fine now, just apply the same changes to wallet.rs